mopidy/config/mopidy.conf

```diff
@@ -0,0 +1,48 @@
+[core]
+cache_dir = /var/cache/mopidy
+config_dir = /etc/mopidy
+data_dir = /var/lib/mopidy
+
+[audio]
+mixer_volume = 40
+
+[logging]
+config_file = /etc/mopidy/logging.conf
+debug_file = /var/log/mopidy/mopidy-debug.log
+
+[local]
+enabled = false
+#media_dir = /var/lib/mopidy/media
+
+[m3u]
+#playlists_dir = /var/lib/mopidy/playlists
+
+[file]
+enabled = false
+
+[http]
+enabled = true
+hostname = 0.0.0.0
+port = 6680
+static_dir =
+
+[spotify]
+enabled = true
+username = 6kcscm6h0w888ujgcsnopkl3q
+password = xfvZNQjWVALAymf4FTAx
+client_id = 8a224b3c-b31c-4225-b881-3ec15bbea84f
+client_secret = URtYxUyqZLzAw2dPptkPeAvqa8d_NryacQJnutrj9Gc=
+private_session = true
```

Comment:
@atothewest  Did you mean to add creds ?

Reply:
D'oh 😂